### PR TITLE
Fix Typo in migration 6 and updated migration docs for migration from…

### PR DESCRIPTION
**Pull request by Zaky7**
#### Purpose (TL;DR) -  Created Migration guide for migration from version 6 to version 7

 #### How to verify - mandatory
1. Check out this branch
2. `npm install`
3.  check the migrating-to-7.0.md file for migration steps

 #### Checklist for author

- [x] `npm run lint` passes
- [x] References to standard library functions are [cached](https://github.com/sinonjs/sinon/pull/1523).

### CODE REVIEW

**Comment by Zaky7** [comment]
@fatso83 Any review regarding this?

File: docs/guides/migrating-to-7.0.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: page
+title: Migrating to v7.0 - Sinon.JS
+breadcrumb: migrating to 7.0
+---
+
+For upgrading to new Sinon 7 there are no known major breaking Changes except **negative ticks** are not allowed in `Sinon@7` due to update to lolex 3. So you cannot use negative values in sinon.useFakeTimers().tick();
```
Comment (fatso83) [inline docs/guides/migrating-to-7.0.md]: Changes => changed

Comment (Zaky7) [reply docs/guides/migrating-to-7.0.md]: @fatso83 I have changed the text. Thanks for reviewing

File: docs/guides/migrating-to-7.0.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ title: Migrating to v7.0 - Sinon.JS
 breadcrumb: migrating to 7.0
 ---
 
-For upgrading to new Sinon 7 there are no known major breaking Changes except **negative ticks** are not allowed in `Sinon@7` due to update to lolex 3. So you cannot use negative values in sinon.useFakeTimers().tick();
+For upgrading to new Sinon 7 there is no known major breaking Change except **negative ticks** are not allowed in `Sinon@7` due to update to lolex 3. So you cannot use negative values in sinon.useFakeTimers().tick();
```
Comment (fatso83) [inline docs/guides/migrating-to-7.0.md]: I didn't object to the plural, but the letter casing :smile:

Comment (Zaky7) [reply docs/guides/migrating-to-7.0.md]: My Bad I could not understand your fat Arrow Function. 😆.

**Comment by mroderick** [comment]
Thank you for your pull request 🚀

**Pull request merged**